force-app/main/default/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -440,7 +440,9 @@
             hh.BillingState = addr.MailingState__c;
             hh.BillingPostalCode = addr.MailingPostalCode__c;
             hh.BillingCountry = addr.MailingCountry__c;
-            hh.Undeliverable_Address__c = addr.Undeliverable__c === undefined ? false : addr.Undeliverable__c;
+            if(hh.hasOwnProperty('Undeliverable_Address__c') && addr.hasOwnProperty('Undeliverable__c')){
+                hh.Undeliverable_Address__c = addr.Undeliverable__c === undefined ? false : addr.Undeliverable__c;
```

Comment:
Does this need to be set to a default value, or is leaving the value _as is_ the goal?

Reply:
In the case where we don't have permissions, this attribute won't exist on the object, so not changing it felt like the best approach.

Reply:
Makes sesnse.

Reply:
```suggestion
                hh.Undeliverable_Address__c = !!addr.Undeliverable__c;
```